src/controller/amapadmin/Volunteers.hx

```diff
@@ -15,6 +16,11 @@ class Volunteers extends controller.Controller
 		checkToken();
 
 		var form = new sugoi.form.Form("msg");
+		// new
+		form.addElement( new Checkbox("allowInformationNotifs", t._("Authorize the sending of information emails to volunteers"), app.user.getGroup().flags.has(db.Group.GroupFlags.AllowInformationNotifs)));
+		form.addElement( new Checkbox("allowAlertNotifs", t._("Authorize the sending of alert emails to free volunteers"), app.user.getGroup().flags.has(db.Group.GroupFlags.AllowAlertNotifs)));
```

Comment:
Je pense qu'il faudrait insérer la checkbox ligne 32 afin que la notif à laquelle elle se rapporte soit claire

Reply:
si possible il était prévu que la désactivation entraine le "grisage" ou la désactivation des champs concernés

Reply:
Noté, je m'en occupe !